apps/android_rpc/app/src/main/jni/Application.mk

```diff
@@ -23,3 +23,7 @@ ifeq ($(USE_VULKAN), 1)
     APP_CPPFLAGS += -DTVM_VULKAN_RUNTIME=1
     APP_LDFLAGS += -lvulkan
 endif
+
+ifeq ($(USE_SORT), 1)
+	APP_CPPFLAGS += -DUSE_SORT=1
```

Comment:
let's use 4-space instead

Reply:
whoops, replaced with spaces.

Reply:
Also split off this commit from Tianqi's changes.